reproman/resource/session.py

```diff
@@ -523,6 +521,40 @@ def _parse_envvars_output(self, out):
             output[split[0]] = split[1]
         return output
 
+    def _prefix_command(self, command, env=None, cwd=None, with_shell=True):
+        """Wrap the command in a shell call with ENV vars and CWD prefixed
+        statment to command. Will pass through the command unchanged if env
+        and cwd are None
+
+        Parameters
+        ----------
+        command : string or list
+            shell command to be run
+        env : dict
+            ENV vars to prefix to command
+        cwd : string
+            working directory to cd into befoe the command is executed
+        with_shell : boolean
+            if True, a "/bin/sh -c" call will wrap the cwd, env and command
+
+        Returns
+        -------
+        string
+            command with env and cwd settings prefixed
+        """
+        prefix = ""
+        if env:
+            prefix = " ".join(['export %s=%s &&' % k for k in env.items()])
```

Comment:
* For consistency, should use `.format`
 * Don't we only need one `export` (i.e. doesn't `export` support multiple k=v pairs)?
 * Do we need to `shlex_quote` the values?